### PR TITLE
Cron tasks of labsync/labcheck are not setup for HA

### LINKED ISSUE #288: The cron tasks of labsync and labcheck won't setup when HA
Thank you for reporting bug!

### Describe the bug
The cron tasks of labsync and labcheck won't setup when HA or switching node role

### Related Job information
If the bug is related a CI job, put the job information below:
* job name: NA
* job build log link: NA
* error log piece: NA

### Steps to reproduce the issue
Describe how to reproduce the issue: NA

### PR DESCRIPTION
**Pull request by huangtianhua**
The cron tasks of labsync and labcheck are not setup
correctly when HA and switching node role.

This changes theree points:
1.rename the playbook conf-new-slave.yaml to conf-cluster.yaml 
2. setup cron tasks of labsync and labcheck when conf-cluster after ha and  when switch role
3. remove the duplicate code for conf-cluster.yaml, just import playbook of 'conf-new-zookeeper.yaml'

Closes: theopenlab/openlab#288

### CODE REVIEW

File: playbooks/switch_role.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- import_playbook: sync-config-files.yaml
+  when: labsync_enabled|default(false)|bool
+- import_playbook: check-nodepool-periodic.yaml
+  when: labcheck_enabled|default(false)|bool
```
Comment (wangxiyuan) [inline playbooks/switch_role.yaml:4]: when switch role, the cron task in the old master node should be cleaned. Has your playbook contained this part?

Comment (huangtianhua) [reply playbooks/switch_role.yaml:4]: Yes, the playbook will setup the task in master nodes and delete the task in slave nodes

Comment (wangxiyuan) [reply playbooks/switch_role.yaml:4]: ah, I find it.

**Review by wangxiyuan — APPROVED** [review]

File: playbooks/switch_role.yaml (hunk 1 above)
Comment (Yikun) [reply playbooks/switch_role.yaml:4]: @wangxiyuan Yes
The labsync will tear down the crontab in nodepool-launcher-slave:zuul-scheduler-slave [1], and the labcheck will tear down the crontab in nodepool-launcher-slave. [2]

[1] https://github.com/theopenlab/labkeeper/blob/master/playbooks/sync-config-files.yaml#L34-L44
[2] https://github.com/theopenlab/labkeeper/blob/master/playbooks/check-nodepool-periodic.yaml#L13-L20

**Comment by huangtianhua** [comment]
recheck

**Comment by huangtianhua** [comment]
recheck

**Comment by bzhaoopenstack** [comment]
recheck

**Comment by bzhaoopenstack** [comment]
recheck

**Review by Yikun — APPROVED** [review]
LGTM

**Review by kiwik — COMMENTED** [review]
Have to change name `conf-new-slave.yaml`

**Review by wangxiyuan — APPROVED** [review]

**Pull request merged**